src/add-search-spec.js

```diff
@@ -71,11 +71,13 @@ const initSearchConfig = (config: any, siteName: string): any => {
 
 describe('Search Results', () => {
   let files: string[];
+  let internalOnlyContent: string[] = [];
 
   function removeUnnecessaryElements() {
     Array.from(
       document.querySelectorAll(
-        '.stache-sidebar, .stache-breadcrumbs, .stache-table-of-contents, stache-hide-from-search'
+        '.stache-sidebar, .stache-breadcrumbs, .stache-table-of-contents, stache-hide-from-search,' +
+        'stache-internal, skyux-restricted-view, .skyux-restricted-view'
```

Comment:
Think we can remove the `stache-internal`?  Also does the `skyuxRestrictedView` directive add the wrapping `skyux-restricted-view` component?

Reply:
Yeah I can remove `stache-internal`. 

For the directive I am adding a class to the directive element `.skyux-restricted-view`. 

The component itself will be captured by just `skyux-restricted-view`